django_project/vota/views/ballot.py

```diff
@@ -6,7 +6,8 @@
 import logging
 from braces.views import LoginRequiredMixin, StaffuserRequiredMixin
 from django.core.urlresolvers import reverse
-from django.views.generic import DetailView, CreateView, DeleteView, UpdateView, ListView
+from django.views.generic import DetailView, \
```

Comment:
Use bracketed syntax for multiline imports please (see coding standards). i.e.

```
from django.views.generic import (
    foo, bar)
```

When all imports cannot fit in second line then do 

```
from django.views.generic import (
    foo, 
    bar,
    etc.)
```